jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/HBaseIntegrationTest.java

```diff
@@ -120,6 +120,7 @@ protected MetricsVerifier createMetricsVerifier() {
             "hbase.region_server.queue.length",
             metric ->
                 metric
+                    .logAttributes()
```

Comment:
This is something that should be done automatically.  Maybe just call it inside hasDataPointsWithAttributes() ?

Reply:
Nah, this design assumed that you'd only want to turn this on in specific cases for certain metrics or whatever. Doing this automatically for every metric assertion would be too verbose.